### DISCUSSION TITLE
proje4 örnek çıktı hatalı mı?

### DISCUSSION THREAD

**Discussion by umutcanclkk** [gusanmaz/NKU_DS_Course_2022#14] [Q&A] [answered]
PreOrder (9  -100  4  12  45  50  75  100 ) output hatalı olduğunu düşünüyorum.

output (9 4 -100 45 12  100 75 50) böyle olduğunu düşünüyorum bilginize arz ederim.
Şu ağaç için
![ağaç](https://user-images.githubusercontent.com/102092541/168308120-4247a1fc-d09e-4659-9d42-df3d8259c9be.png)

**Comment by gusanmaz** [comment, marked as answer]
Evet, hatalı çıktı var. Teşekkürler. Mobilden düzenleme yapamıyorum. Eve gidince proje aciklamalarini guncelleyecegim.

**Reply by gusanmaz** [reply]
Proje.md ve _BMB212_BST_ template reposundaki Readme.md uzerinde gerekli degisiklik yapildi.
